sbi/inference/snpe/snpe_a.py

```diff
@@ -142,7 +143,8 @@ def train(
             resume_training: Can be used in case training time is limited, e.g. on a
                 cluster. If `True`, the split between train and validation set, the
                 optimizer, the number of epochs, and the best validation log-prob will
-                be restored from the last time `.train()` was called.
+            force_first_round_loss: If `True`, train with maximum likelihood,
```

Comment:
make this more informative for users, e.g., "If `True`, train with maximum likelihood loss in every round, i.e., potentially ignoring the correction for using a proposal distribution different from the prior."